src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -139,7 +139,7 @@
   attachmentFileContainer: {
     backgroundColor: COLORS.gray,
     justifyContent: 'center',
-    minHeight: dimen.normalizeDimen(64),
+    height: dimen.normalizeDimen(64),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The minHeight property has been replaced with a fixed height. This could potentially cut off content if it exceeds the specified height.

```diff
-    minHeight: dimen.normalizeDimen(64),
+    height: dimen.normalizeDimen(64),
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->